src/Compiler/Scheme/Racket.idr

```diff
@@ -119,6 +119,10 @@ data Done : Type where
 cftySpec : FC -> CFType -> Core String
 cftySpec fc CFUnit = pure "_void"
 cftySpec fc CFInt = pure "_int"
+cftySpec fc CFUnsigned8 = pure "_int8"
+cftySpec fc CFUnsigned16 = pure "_int16"
+cftySpec fc CFUnsigned32 = pure "_int32"
+cftySpec fc CFUnsigned64 = pure "_int64"
 cftySpec fc CFUnsigned8 = pure "_uint8"
```

Comment:
This ones should have been the new CFIntX variants: #1462

Reply:
Good catch. How come we did not get a warning about unreachable clauses?